test/study-webidl.js

```diff
@@ -34,13 +54,14 @@ interface Valid {};
 [Global=Window,Exposed=*]
 interface Invalid;
 `);
-    assert.deepEqual(report[0]?.name, 'invalid');
-    assert.deepEqual(report[0].message, `Syntax error at line 3, since \`interface Invalid\`:
+    assertEqual(report, 'length', 1);
```

Comment:
```suggestion
    assert.equal(report.length, 1);
```
I don't think assertEqual helps here

Reply:
It does because it will report the entire anomaly report to the console, which in turns allows to quickly understand why the assumption was incorrect. Without that, you would just get an "expected 1, got 2" message and you'd need to reproduce the run locally (if error occurred in CI) and switch to debug mode.

Reply:
then I suggest an assertReportLength or something, rather than re-using the already somewhat convoluted assertEqual